.github/workflows/build_navitia_packages_for_dev.yml

```diff
@@ -1,6 +1,8 @@
 name: Build Navitia Packages For Dev
 
 on:
+  # for test, TO BE REMOVED BEFORE MERGE
+  pull_request:
```

Comment:
Sure you want to keep those line ?

Reply:
NO, I'll remove them when the PR is approved.

Reply:
But if I approve, the PR will get merged by mergify :/

Reply:
dead lock in progress :lock: :skull: :smile:

Reply:
that's why i put a `do_not_merge` :D